orchestrators/ProcurementOperations/src/Services/GrIrAccrualService.php

```
@@ -134,126 +106,46 @@ public function matchWithInvoice(
         string $writeOffReason,
         string $writeOffBy,
         ?string $writeOffAccountId = null,
-        ?string $approvalReference = null,
     ): GrIrAccrualData {
-        if ($accrual->status !== 'OPEN') {
-            throw new \InvalidArgumentException(
-                "Cannot write off accrual {$accrual->accrualId}: status is {$accrual->status}"
-            );
-        }
+        $accrual = $this->getAccrual($accrualId);
+        if ($accrual === null) { throw new \InvalidArgumentException("Accrual not found: {$accrualId}"); }
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '138', column '44'). (undefined)

(MissingImport)